test/regress/regress1/sygus-abduct-test-user.smt2

```diff
@@ -0,0 +1,28 @@
+; COMMAND-LINE: --produce-abducts
```

Comment:
Is it possible, with this new architecture, to include also `--check-sygus-sol`?

Reply:
Yes, this should work as expected.